### PR TITLE
feat(docs): serverless document

### PR DESCRIPTION
**Pull request by azuregaya**

### CODE REVIEW

File: en_US/deployments/view_deployment.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+:::: tabs type:card
+::: tab Serverless 
+## View deployment information
+The Deployment overview page provides access to the status and connectivity information.
+
+![overview](./_assets/serverless_overview.png)
+
+
+- Status: The deployment running status and creation time.
+- Number of sessions: Concurrent sessions and maximum number of sessions, during Beta, the maximum is 100.
```
Comment (LenaLenaPan) [inline en_US/deployments/view_deployment.md:15]:
```suggestion
- Number of sessions: Concurrent sessions and maximum number of sessions, a maximum of 100 sessions are supported in the Beta version.
```

File: en_US/deployments/view_deployment.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+
+:::: tabs type:card
+::: tab Serverless 
+## View deployment information
+The Deployment overview page provides access to the status and connectivity information.
+
+![overview](./_assets/serverless_overview.png)
+
+
+- Status: The deployment running status and creation time.
```
Comment (LenaLenaPan) [inline en_US/deployments/view_deployment.md:14]:
```suggestion
- Status: Deployment running status and creation time.
```

File: en_US/deployments/view_deployment.md [hunk 3]
```diff
@@ -1,6 +1,27 @@
 # Deployment Overview
 
-In this tutorial, you will learn how to view deployment information, obtain connection addresses, view API addresses, etc.
+In this tutorial, you will learn how to view deployment information, obtain connection addresses, view port, etc.
```
Comment (0721Betty) [inline en_US/deployments/view_deployment.md:3]: connection addresses, view port -> connection address, view ports

File: en_US/deployments/view_deployment.md [hunk 4]
```diff
@@ -1,6 +1,27 @@
...
+
+
+:::: tabs type:card
+::: tab Serverless 
+## View deployment information
+The Deployment overview page provides access to the status and connectivity information.
```
Comment (0721Betty) [inline en_US/deployments/view_deployment.md:9]: connectivity -> connection

File: en_US/intro/restriction.md [hunk 5]
```diff
@@ -1,12 +1,33 @@
-# Use limitations
+# limitations
```
Comment (0721Betty) [inline en_US/intro/restriction.md:1]: Limitations

File: en_US/rule_engine/introduction.md [hunk 6]
```diff
@@ -13,6 +13,10 @@ In EMQX Cloud, using Data Integrations has the following requirements:
   - The resource technically only supports internal network access, so you need to configure [VPC peering](../deployments/vpc_peering.md) before creating the resource and open the security group.
   - If public access is needed in a professional environment, [NAT Gateway](../vas/nat-gateway.md) can offer the abilities.
 
+::: tip
+Serverless deployments doesn't support Data Integrations.
```
Comment (lucas-emqx) [inline en_US/rule_engine/introduction.md:17]: doesn't -> don't

File: en_US/quick_start/create_serverless.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
+# Create Serverless (Beta) deployment
+
+You can create a Serverless (Beta) deployment in a few easy steps.
+
+Currently only one Serverless(Beta) deployment can be created under each account. Until 2023.4.1, Serverless deployments are limited to a maximum of 100 sessions in Beta testing and are completely free to use.
```
Comment (lucas-emqx) [inline en_US/quick_start/create_serverless.md:5]: Missing space in "Serverless(Beta)"

File: en_US/price/calculator.md [hunk 8]
```diff
@@ -1,35 +1,49 @@
...
+- For users who are not logged in, you can go to [pricing page](https://www.emqx.com/en/cloud/pricing) to check the available plans.
 
-  ![pricing](./_assets/pricing.png)
+We offer a Serverless price calculator so you can estimate your approximate monthly spending based on your business.
 
-- For users who are **registered** to EMQX Cloud, you can go to [Price Estimate](https://cloud-intl.emqx.com/console/deployments/0?oper=new) to select plan, use price estimation and create a deployment.
+![calculator](./_assets/calculator_serverless.png)
 
-  ![calculator](./_assets/calculator.png)
+You can check the price of Dedicated plas by selecting the specification.
```
Comment (lucas-emqx) [inline en_US/price/calculator.md:15]: plas -> plan

**Review by luoyq0629 — APPROVED** [review]

**Review by lucas-emqx — APPROVED** [review]

**Review by 0721Betty — APPROVED** [review]

**Pull request merged**